app/index.html

```diff
@@ -84,6 +84,7 @@
     <script src="vendor/ng-http-rate-limiter/dist/ng-http-rate-limiter.js?v=$DIM_VERSION"></script>
     <script src="vendor/angular-translate/angular-translate.min.js?v=$DIM_VERSION"></script>
     <script src="vendor/idb-keyval/dist/idb-keyval-min.js?v=$DIM_VERSION"></script>
+    <script src="vendor/angular-local-storage/dist/angular-local-storage.min.js"></script>
```

Comment:
Localstorage is a string-based storage.  This will automatically preserve the type when interacting with the store.  I didn't like having to use parseInt/stringify every time I touched localstorage so this will take care of that.  It also allows for prefixed-naming that avoids collisions but we're not too worried about that.